pkgs/top-level/all-packages.nix

```diff
@@ -272,6 +272,8 @@ with pkgs;
 
   ast-grep = callPackage ../development/tools/misc/ast-grep { };
 
+  alt-server = callPackage ../tools/misc/alt-server { };
```

Comment:
The letter `l` comes before the letter `s`. Please update.

Reply:
I believed this was fixed in [this](https://github.com/NixOS/nixpkgs/pull/246161/commits/70972f26e0ecdd586bc94fdd01d7167a93431bb7) commit, but I seem to have been mistaken, could you explain what you mean?